apps/web/jest.config.ts

```diff
@@ -7,7 +7,7 @@ const config: Config.InitialOptions = {
   verbose: true,
   roots: ["<rootDir>"],
   setupFiles: ["<rootDir>/test/jest-setup.js"],
-  testMatch: ["**/test/lib/**/*.(spec|test).(ts|tsx|js)"],
+  testMatch: ["**/test/lib/**/*.(spec|test).(ts|tsx|js)", "**/__tests__/**/*.(spec|test).(ts|tsx|js)"],
```

Comment:
This adds functionality so when we create test next to our components we can use this folder structure.
- components/
   - myComponent/
       - index.tsx
       - __test__/
          - myComponent.test.ts

Reply:
How about components that don't have this structure right now?

Can we use:
- MyComponent.tsx
- MyComponent.test.tsx

in the same level?